Extension/lib/filter/rules/simple-regex.js

```diff
@@ -32,7 +32,7 @@
 
             regexAnySymbol: ".*",
             regexSeparator: "([^ a-zA-Z0-9.%]|$)",
-            regexStartUrl: "^(http|https|ws|wss)://([a-z0-9-_.]+\\.)?",
+            regexStartUrl: "^(http://|https://|ws://|wss://|stun:|stuns:|turn:|turns:)([a-z0-9-_.]+\\.)?",
```

Comment:
Wow, that is a rather complicated regex.

First of all, we can change `stun:` to `stun://`, etc.

Then we'll be able to change this expression into something more readable:
`^.{2,5}://([a-z0-9-_.]+\\.)?`